scripts/acceptance

```diff
@@ -24,7 +24,9 @@ then
   echo "=============================="
   echo "Running Test on your machine"
   echo "=============================="
+  export TEST_ENVIRONMENT_ROOT_URL=http://localhost:3000
```

Comment:
Can this not be injected into the docker-compose.override.yaml  similar to `scripts/test` and `docker-compose.override.yaml`?

https://github.com/DEFRA/ffc-sfi-apply-web/blob/main/scripts/test
https://github.com/DEFRA/ffc-sfi-apply-web/blob/main/docker-compose.override.yaml

Reply:
We could do this actually. Running test:local does not build any docker asset, it simply fires up a browser.
However you are right, I will have a rethink of this. 👍